Dockerfile

```diff
@@ -117,20 +109,10 @@ RUN echo "" >> ~/.bashrc && \
 
 # Install Composer
 RUN curl -sSL https://getcomposer.org/installer | php -- --filename=composer --install-dir=/usr/bin
-RUN echo 'export PATH="$HOME/.composer/vendor/bin:$PATH"' >> ~/.bashrc
-
-# Copy the php-cs-fixer rules inside the docker container
-COPY ./configfiles/.php_cs /root/configfiles/.php_cs
```

Comment:
Could you explain this change?

Reply:
It would be better (in my opinion) to add this file to the project repository.